express/src/controllers/sign-in.ts

```diff
@@ -47,14 +48,18 @@ export const processLoginOtpMobile = async function(req: Request, res: Response)
     try {
         response = await cognitoClient.respondToMfaChallenge(req.session.cognitoUser?.Username as string, req.body['sms-otp'], req.session.session as string);
     } catch (error) {
-        if(error instanceof CodeMismatchException) {
+        if (error instanceof CodeMismatchException) {
             const errorMessages: Map<string, string> = new Map<string, string>();
             errorMessages.set('smsOtp', 'The code you entered is not correct or has expired - enter it again or request a new code')
             res.render('common/check-mobile.njk', {
                 mobileNumber: req.session.mobileNumber,
                 formActionUrl: "/sign-in-otp-mobile",
                 errorMessages: errorMessages
             })
+            return;
+        }
+        else {
+            throw error;
         }
```

Comment:
This perhaps can be handled by some middleware too to move out exception handling out of the controllers?